src/com/esotericsoftware/yamlbeans/YamlIterator.java

```diff
@@ -0,0 +1,7 @@
+package com.esotericsoftware.yamlbeans;
+
+public interface YamlIterator<E> {
```

Comment:
Can we use java.util.Iterator instead of our own class?

Reply:
Modified according to your suggestion